src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -37,5 +37,59 @@ public IActionResult MultipleAnswerQuestionForm(string id, int[] answers)
             };
             return View(question);
         }
+
+        public IActionResult EditMultipleAnswerQuestion(Guid id)
+        {
+            var question = new MultipleAnswerQuestionViewModel()
+            {
+                SourceQuestion = questionRepository.PickById(id),
+                // ChosenOptions = answers
+            };
+            return View(question);
+        }
+
+        public IActionResult AddMultipleAnswerQuestion()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult AddMultipleAnswerQuestion([FromBody] MAQDTOModel fromAjax)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            for (int i = 0; i < fromAjax.jTextList.Length; i++)
+            {
+                allAnswers.Add(new MultipleAnswerQuestionOption(fromAjax.jTextList[i], fromAjax.jCheckboxList[i]));
+            }
+            questionRepository.MethodForAdd(new MultipleAnswerQuestion(fromAjax.questionText, allAnswers));
+
+            return View(); 
+            
+        }
+        [HttpPost]
+        public IActionResult EditMultipleAnswerQuestion( int[] editAnswers, string[] answersText, string questionText, Guid Id)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            int g = 0;
+            bool isRight;
+            for (int i = 0; i < answersText.Length; i++)
+            {
+                if (editAnswers[g] == i)
+                {
+                    isRight = true;
+                    g++;
+                }
+                else
+                {
+                    isRight = false;
+                }
+                allAnswers.Add(new MultipleAnswerQuestionOption(answersText[i], isRight));
+            }
+            MultipleAnswerQuestion question = questionRepository.PickById(Id);
+            question.UpdateQuestion(questionText, allAnswers);
+            questionRepository.SaveAllChanges();
+            return View();
+        }
```

Comment:
I get an exception when I try to edit a question.
![image](https://user-images.githubusercontent.com/1754953/100545616-bede5d00-326d-11eb-8258-8f31f7bb0a72.png)